apps/docs-snippets/src/guide/encoding/working-with-bytes.test.ts

```diff
@@ -0,0 +1,143 @@
+import { randomBytes } from 'crypto';
+import {
+  ArrayCoder,
+  B256Coder,
+  B512Coder,
+  BigNumberCoder,
+  BooleanCoder,
+  EnumCoder,
+  NumberCoder,
+  RawSliceCoder,
+  StdStringCoder,
+  StringCoder,
+  StructCoder,
+  TupleCoder,
+  VecCoder,
+  hexlify,
+} from 'fuels';
+
+/**
+ * @group node
+ */
+describe('working with bytes', () => {
```

Comment:
I actually prefer the native -> automatic -> heap grouping, but I was trying to intentionally differentiate between types and byte structure. I don't think there is much value in a snippet each for the automatic and heap coders because the structure is essentially the same, apart from the underlying native coders that make up the child coders of the parent. 

However I can appreciate we don't want it to be cluttered. I've added some spacing to the tests, does that help?